debian/postinst

```diff
@@ -26,13 +26,27 @@ SYSTEMD_HELPER_ENABLED_AUTO_ATTACH_DSH="/var/lib/systemd/deb-systemd-helper-enab
 SYSTEMD_HELPER_ENABLED_WANTS_LINK="/var/lib/systemd/deb-systemd-helper-enabled/multi-user.target.wants/ua-auto-attach.service"
 
 
+check_esm_infra_is_installed() {
+    expected_output="enabled"
+    actual_output=$(ua status | grep esm-infra | awk '{print $3}')
```

Comment:
How's this look just in case we end up changing status column order for some reason.
```suggestion
    actual_output=$(ua status | egrep 'esm-infra.*enabled)
```